kiva/fonttools/font_manager.py

```diff
@@ -74,17 +74,18 @@ class FontManager:
     # Increment this version number whenever the font cache data
     # format or behavior has changed and requires a existing font
     # cache files to be rebuilt.
-    __version__ = 9
+    __version__ = 10
 
     def __init__(self, size=None, weight="normal"):
         self._version = self.__version__
 
         self.__default_weight = weight
         self.default_size = size if size is not None else 12.0
-
-        paths = []
+        self.default_family = "sans-serif"
```

Comment:
This is basically equivalent to the previous value. Bitstream Vera and Helvetica are both sans-serif families. But since not every system may have them, we can use the generic family name for the default and get good results thanks to the `preferred_fonts` dictionary listing several options for sans-serif.